lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
Instead of replacing the entire file, can't we just redefine the constants? (I suppose to avoid any sort of warning message, we could use a global variable instead of a constant in core).